src/runtime/HalideRuntime.h

```diff
@@ -685,25 +685,23 @@ typedef enum halide_target_feature_t {
 
     halide_target_feature_user_context = 24,  ///< Generated code takes a user_context pointer as first argument
 
-    halide_target_feature_register_metadata = 25,  ///< Generated code registers metadata for use with halide_enumerate_registered_filters
+    halide_target_feature_matlab = 25,  ///< Generate a mexFunction compatible with Matlab mex libraries. See tools/mex_halide.m.
```

Comment:
I could go either way as well. If/when we decide to avoid recycling feature numbers, then the can_use_feature runtime call will need attention (it assumes all features will fit into a 64-bit mask at present).